test/e2e/app-dir/app-typescript/app/inner/page.js

```diff
@@ -0,0 +1,11 @@
+export default function Page() {
+  return <p>hello</p>
+}
+
+// export const not_used = 1
+
+// export const dynamic = 'auto'
```

Comment:
The type error cases are not tested by that, are them?

Reply:
Yeah these are comments for debugging.